packages/replay-next/src/contexts/FocusContext.tsx

```diff
@@ -88,22 +98,18 @@ export function FocusContextRoot({ children }: PropsWithChildren<{}>) {
   // to indicate that what's currently being showed is stale.
   const [isTransitionPending, startTransition] = useTransition();
 
-  // Refine the loaded ranges based on the focus window.
   useEffect(() => {
-    if (rangeForDisplay === null || rangeForDisplay === initialRange) {
+    if (!requestedRange) {
```

Comment:
So this change means that we'll always call `client.requestFocusWindow` when this context mounts. I assume that's an intentional part of the changes here just wanted to verify.

Reply:
> So this change means that we'll always call client.requestFocusWindow when this context mounts.

I don't think it does because `requestedRange` is initially `undefined`.